client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/pages/participant-detail-manage/components/participant-create-wizard/participant-create-wizard.component.html

```diff
@@ -136,7 +136,7 @@ <h3>{{ 'Suitable accounts found' | translate }}</h3>
                     [patchFormValueFn]="patchFormValueFn"
                     [shouldEnableFormControlFn]="shouldEnableFormControlFn"
                     [user]="user"
-                    (changeEvent)="personalInfoFormValue = $event"
+                    (changeEvent)="personalInfoFormValue = $event; updateLockout()"
```

Comment:
I would prefer just calling one method per attribute.

Reply:
Use updateByValueChange($event) for both.